Jenkinsfile

```diff
@@ -101,6 +102,23 @@ node('ubuntu-zion') {
         gitHub.statusUpdate commitId, 'success', 'test', 'Tests succeeded'
       }
     }
+    stage('Evaluate') {
+      
+      //Create tar of our image
+      OsTools.runSafe(this, "docker save ${imageName} -o ${env.WORKSPACE}/${tarName}")
+      
+      //decide which stage we are creating
```

Comment:
Something we should think about: Do we really want to fail the docker image build if something in the base image (as opposed to the app itself) violates policies? At least initially, when quite a bit of catchup/cleanup might be in order, this seems to risk not being able to deliver a docker image for some time.

Reply:
Anecdotally, RedHat (as I've been told) treats base image violations very seriously due to some of their customers who use it and historically pushes UBI fixes very quickly. In my opinion, I think we should not fail the build until we see how bad the violation occurrence is.

Reply:
The alternative would be to ship a known vulnerable package, I think I'm okay with a delayed image. Should we ship something we know has security concerns? If you check out the latest scan, it doesn't scan the IQ jar or the CLI jar because they are uber jars.

Reply:
>  I think I'm okay with a delayed image

Please run this by and approved by the larger leadership team. In the past, folks had a strong desire to not call a release done until the docker images were also updated. I don't want us tackling the next release and then folks coming out of the bushes asking why the docker images isn't available for days, or even weeks, until everything wrong with the base image is resolved.

>  Should we ship something we know has security concerns?

This might be a good use of IQ's grandfathering feature, take the status quo as baseline, improve upon on it as resources allow, instead of stopping everything.

Reply:
I discussed the idea of using Grandfathering with Mike and Brian and they both agreed that that is a good approach to this scenario. Therefore, I shall do that.

Reply:
Also discussed blocking the build, we all agreed we should block the push if there is an issue that is found during the release process. We hope that continuous monitoring will prevent said scenario.